src/framework/COScript.h

```diff
@@ -13,6 +13,7 @@
 @class COScript;
 
 @protocol COPrintController
+- (void)scriptHadException:(NSException*)exception;
```

Comment:
I do find the naming of this a bit odd within Cocoa. How about instead `scriptThrewException:`, or `scriptEncounteredException:`? "Had" feels odd within Cocoa norms.